coursefunctions.h

```diff
@@ -172,30 +220,50 @@ void TestDrawTriangle(Buffer2D<PIXEL> & target)
         verts[2] = {50, 452, 1, 1};
         PIXEL colors1[3] = {0xffff0000, 0xffff0000, 0xffff0000};
         // Your color code goes here for 'attr'
+        // Loop to add colors to attr
+        for (int  i = 0; i < 3; ++i)
+        {
```

Comment:
Instead of doing a loop here it would be more efficient to just make assignments.

Reply:
I would be very surprised if you saw a dip in speed because of these loops. I think this is a more universal solution than just assignments.